### PR TITLE
Link support osu! to new store page

### LINKED ISSUE #1634: [/home/support] Support osu! links are still redirecting to the old page

### PR DESCRIPTION
**Pull request by notbakaneko**
fixes #1634 

`STORE_SUPPORTER_TAG_ID=` needs to be set in `.env` since it's variable.

### CODE REVIEW

File: .env.example [hunk 1]
```diff
@@ -101,6 +101,7 @@ PUSHER_SECRET=
 PAYMENT_SANDBOX=true
 
 STORE_NOTIFICATION_CHANNEL=test
+STORE_SUPPORTER_TAG_ID=10 # Your id will vary.
```
Comment (nanaya) [inline .env.example]: `_PRODUCT_ID`?

**Comment by notbakaneko** [comment]
I'm thinking it may be worth having matching routes for products with a `custom_class`?

**Comment by nanaya** [comment]
probably

File: routes/web.php [hunk 2]
```diff
@@ -212,6 +211,12 @@
             'CheckoutController',
             ['only' => ['index', 'store']]
         );
+
+        Route::resource(
+            'product',
```
Comment (nanaya) [inline routes/web.php]: plural?

Comment (notbakaneko) [reply routes/web.php]: the current route is singular :(

Comment (nanaya) [reply routes/web.php]: replace it? :D

File: app/helpers.php [hunk 3]
```diff
@@ -985,3 +985,8 @@ function group_users_by_online_state($users)
         'offline' => $offline,
     ];
 }
+
+function product_url($product)
+{
+    return route('store.product.show', ['product' => $product]);
```
Comment (nanaya) [inline app/helpers.php]: `route('store.products.show', $products)` _should_ work so this helper probably isn't needed.

**Comment by nanaya** [comment]
may want to search for `/store/product/`

**Pull request merged**